rest-api-spec/src/main/resources/rest-api-spec/api/mget.json

```diff
@@ -16,7 +16,7 @@
         }
       },
       "params": {
-        "fields": {
+        "stored_fields": {
           "type": "list",
           "description" : "A comma-separated list of fields to return in the response"
         },
```

Comment:
Inconsistent with description in get.json - should be:

    A comma-separated list of *stored* fields to return in the response